tests/bsim/bluetooth/host/l2cap/credits_seg_recv/src/common.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
```

Comment:
Several files for the test are using the wrong year :) 
```suggestion
 * Copyright (c) 2023 Nordic Semiconductor ASA
```